src/views/scans/showScansModal.tsx

```diff
@@ -88,6 +90,7 @@ const ShowScansModal: React.FC<ShowScansModalProps> = ({
       title={t('view.label', { context: 'scans-names', name: scan?.name })}
       isOpen={isOpen}
       onClose={() => onClose()}
+      {...(actions && { actions })}
```

Comment:
optional: Since we're already handling `onClose` it would be more efficient to leverage our existing work. Since we're not likely to reuse this component, and it's one less setting outside of the modal itself... 

```
actions={[
        <Button key="close" variant="secondary" onClick={() => onClose()}>
          {t('table.label', { context: 'close' })}
        </Button>
      ]}
```
or if you want to still expose the actions with an internal default... 

```
actions={
        actions || [
          <Button key="close" variant="secondary" onClick={() => onClose()}>
            {t('table.label', { context: 'close' })}
          </Button>
        ]
      }
```